facdb/pipelines.py

```diff
@@ -264,32 +264,71 @@ def nysdec_lands(df: pd.DataFrame = None):
 
 
 @Export
+@Function1B(
+    street_name_field="parsed_sname",
+    house_number_field="parsed_hnum",
+    zipcode_field="zip_code",
+)
+@ParseAddress(raw_address_field="location_address")
 @Prepare
 def nysdec_solidwaste(df: pd.DataFrame = None):
+    df = df[df.county.isin(["New York", "Kings", "Bronx", "Queens", "Richmond"])]
     return df
 
 
 @Export
+@Function1B(
+    street_name_field="street_name",
+    house_number_field="house_number",
+    zipcode_field="zipcode",
+)
 @Prepare
 def nysdoccs_corrections(df: pd.DataFrame = None):
+    df["zipcode"] = df.zipcode.apply(lambda x: x[:5])
+    df = df[df.county.isin(["New York", "Kings", "Bronx", "Queens", "Richmond"])]
```

Comment:
We should also pass in a borough field based on county into function 1B to retain maximal info